composer.json

```diff
@@ -18,7 +18,8 @@
   "require": {
     "php": ">=7.3.0",
     "laravel/framework": ">=5.3.0",
-    "phpunit/phpunit": ">=7.0"
+    "phpunit/phpunit": ">=7.0",
+    "ronasit/laravel-helpers": "~2.1.0"
```

Comment:
it would be better if we can prevent packages dependencies

Reply:
Done